macro/README.md

```diff
@@ -0,0 +1,47 @@
+# Helper macros for development 
+
+This crate provides a few helper macros for development.
```

Comment:
```suggestion
This crate provides a few helper macros for guest program development.
```

Reply:
This is macro is for host only @yoichi-nexus .